renovate.json

```diff
@@ -4,32 +4,47 @@
     "config:recommended"
   ],
   "ignorePaths": [],
-  "labels": ["Skip Changelog", "dependencies"],
+  "labels": [
+    "Skip Changelog",
+    "dependencies"
+  ],
```

Comment:
Can you keep your changes limited to what's actually being done in the PR rather than also adding different style?